config.toml

```diff
@@ -12,6 +12,7 @@ timeout = 300000
     socialProfiles = ["https://twitter.com/ApacheCamel"]
     organizationLogo = "https://camel.apache.org/_/img/logo-d.svg"
     organizationDescription = "Apache Camel ™ is a versatile open-source integration framework based on known Enterprise Integration Patterns. Camel empowers you to define routing and mediation rules in a variety of domain-specific languages, including a Java-based Fluent API, Spring or Blueprint XML Configuration files, and a Scala DSL."
+    token = ""
```

Comment:
Remove this completely, we're using public APIs, until we hit rate limits we don't need this.

Reply:
I have removed the token but the page returns an error[error]. I think it has to do with API rate limit. What do you think?

![error](https://user-images.githubusercontent.com/49795064/78473525-37216c80-7739-11ea-9d34-91cf19167680.PNG)